staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -435,17 +435,15 @@ apiserver_watch_cache_consistent_read_total{fallback="true", resource="pods", su
 				podList.ResourceVersion = tc.watchCacheRV
 				return nil
 			}
-			// TODO: Use fake clock for this test to reduce execution time.
-			cacher, _, err := newTestCacher(backingStorage)
+			c := testingclock.NewFakeClock(time.Now())
+			cacher, _, err := newTestCacherWithoutSyncing(backingStorage, c)
 			if err != nil {
 				t.Fatalf("Couldn't create cacher: %v", err)
 			}
 			defer cacher.Stop()
 			proxy := NewCacheProxy(cacher, backingStorage)
-			if !utilfeature.DefaultFeatureGate.Enabled(features.ResilientWatchCacheInitialization) {
```

Comment:
nit: let's revert this check for now until the feature is not GA

Reply:
this test now uses `newTestCacherWithoutSyncing` so it needs to ensure that the cache is ready. 

unless our assumption is that all the calls will be retried once the feature GAs

Reply:
> this test now uses newTestCacherWithoutSyncing so it needs to ensure that the cache is ready.

ok, right

> unless our assumption is that all the calls will be retried once the feature GAs

not automatically - and rewriting all tests would be really painful...